core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -638,6 +637,9 @@ object KafkaConfig {
   val RequestTimeoutMsDoc = CommonClientConfigs.REQUEST_TIMEOUT_MS_DOC
   val ConnectionSetupTimeoutMsDoc = CommonClientConfigs.SOCKET_CONNECTION_SETUP_TIMEOUT_MS_DOC
   val ConnectionSetupTimeoutMaxMsDoc = CommonClientConfigs.SOCKET_CONNECTION_SETUP_TIMEOUT_MAX_MS_DOC
+  val ProcessRolesDoc = "[ALPHA] The roles that this process plays: 'broker', 'controller', or 'broker,controller' if it is both. " +
```

Comment:
Can we call it "Early Access" instead of Alpha?

Reply:
I think I'm going to keep this as internal for now. Let's reconsider this after we have something that can actually start up.